flocker/node/agents/blockdevice.py

```diff
@@ -167,18 +173,30 @@ class UnattachedVolume(VolumeException):
 
 MOUNT_BLOCK_DEVICE = ActionType(
     u"agent:blockdevice:mount",
-    [VOLUME],
+    [DATASET_ID],
     [],
     u"A block-device-backed dataset is being mounted.",
 )
 
+MOUNT_BLOCK_DEVICE_DETAILS = MessageType(
```

Comment:
I wonder how we could eventually add these useful details to the surrounding ACTION?